### PR TITLE
Adds more makeshift items

### PR DESCRIPTION
**Pull request by nmajask**
# Document the changes in your pull request

Adds more makeshift tools and weapons that are locked behind syndicate crafting, though I do plan on making some of these craftable to crew through skills if that gets merged. Ports/revives ["adds ghetto/improvised tools (port)" #8972](https://github.com/yogstation13/Yogstation/pull/8972), ["Makeshift pistol" #52555](https://github.com/tgstation/tgstation/pull/52555), ["adds the improvised pickaxe + jackhammer weighs less" #50534](https://github.com/tgstation/tgstation/pull/50534) without the weight changes, and tape from ["Refactors embedding, adds harmless embeds, sticky tape, admin buttons to make everything embeddable" #48959](https://github.com/tgstation/tgstation/pull/48959) without the surrounding system. Also adds sounds from ["Adds medical sounds" #21268](https://github.com/Baystation12/Baystation12/pull/21268) by [comma](https://github.com/comma) and various coder sprites.

# Wiki Documentation

Add the tools to the tool page and maybe mention the new craftable items on the syndicate crafting book section of the traitor items page. 

# Changelog

:cl:  boodaliboo, Bucovineanu, Ryll-Ryll, Fikou, nmajask
rscadd: adds new improvised tools, tape, the makeshiftov pistol, its respective magazine an improvised pickaxe, and a makeshift suppressor
/:cl:

### CODE REVIEW

**Comment by maxion12345** [comment]
hell yeah makeshift is a nice buy then

**Comment by bruhlookatthisdood** [comment]
so these will be locked behind syndicate crafting?
there goes my ghetto gun factory plans.

**Review by nesisknesis — COMMENTED** [review]
no name 
image

**Comment by nmajask** [comment]
> no name image

fixed

**Review by nesisknesis — APPROVED** [review]

**Comment by ghost** [comment]
Amogus

File: code/game/objects/items/kitchen.dm [hunk 1]
```diff
@@ -270,4 +271,19 @@
...
+	name = "makeshift knife"
+	icon_state = "knife_makeshift"
+	desc = "A flimsy, poorly made replica of a classic cooking utensil."
+	force = 8
+	throwforce = 8
+
+/obj/item/kitchen/knife/makeshift/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	..()
+	if(prob(5))
+		to_chat(user, "<span class='danger'>[src] crumbles apart in your hands!</span>")
```
Comment (tattax) [inline code/game/objects/items/kitchen.dm]:
```suggestion
		to_chat(user, span_danger("[src] crumbles apart in your hands!"))
```

File: code/game/objects/items/devices/multitool.dm [hunk 2]
```diff
@@ -172,3 +172,22 @@
...
+	name = "makeshift multitool"
+	desc = "As crappy as it is, its still mostly the same as a standard issue Nanotrasen one."
+	icon = 'icons/obj/improvised.dmi'
+	icon_state = "multitool_makeshift"
+	toolspeed = 2
+
+/obj/item/multitool/makeshift/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	..()
+	if(prob(5))
+		to_chat(user, "<span class='danger'>[src] crumbles apart in your hands!</span>")
```
Comment (tattax) [inline code/game/objects/items/devices/multitool.dm]:
```suggestion
		to_chat(user, span_danger("[src] crumbles apart in your hands!"))
```

Comment (tattax) [reply code/game/objects/items/devices/multitool.dm]: consistent

File: code/game/objects/items/devices/multitool.dm [hunk 3]
```diff
@@ -172,3 +172,22 @@
...
+
+/obj/item/multitool/makeshift/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	..()
+	if(prob(5))
+		to_chat(user, span_danger("[src] crumbles apart in your hands!"))
+		qdel(src)
+		return
+	else if(prob(5))
+		user.rad_act(20)
+		to_chat(user, "<span class='userdanger'>[src] breaks down and emits dangerous rays!</span>")
```
Comment (tattax) [inline code/game/objects/items/devices/multitool.dm]:
```suggestion
		to_chat(user, span_userdanger("[src] breaks down and emits dangerous rays!"))
```
forgot this one

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**